huggingface/pytorch/inference/docker/1.13/py3/sdk2.16.1/Dockerfile.neuronx

```diff
@@ -8,6 +8,7 @@ ARG PYTHON=python3.10
 ARG PYTHON_VERSION=3.10.12
 ARG MMS_VERSION=1.1.11
 ARG MAMBA_VERSION=23.1.0-4
+ARG BOTOCORE_VERSION=1.34.94
```

Comment:
nit: could you leave a comment there describing the conflicts between botocore and pip ? Thanks